src/zepben/eas/client/eas_client.py

```diff
@@ -30,45 +30,73 @@ def __init__(
         self,
         host: str,
         port: int,
-        client_id: Optional[str] = None,
+        protocol: str = "https",
         token_fetcher: Optional[ZepbenTokenFetcher] = None,
+        client_id: Optional[str] = None,
         client_secret: Optional[str] = None,
         username: Optional[str] = None,
         password: Optional[str] = None,
-        protocol: str = "https",
-        verify_certificate: bool = False,
+        verify_certificate: bool = True,
         ca_filename: Optional[str] = None,
         session: ClientSession = None,
         json_serialiser = None
     ):
         """
-        :param host: The host string of the Evolve App Server, including the protocol, e.g."https://evolve.local"
+        Construct a client for the Evolve App Server. If the server is HTTPS, authentication may be configured.
+        Authentication may be configured in one of two ways:
+            - Specifying a ZepbenTokenFetcher directly via the token_fetcher parameter
+            - Specifying the client ID of the Auth0 via the client_id parameter, plus one of the following:
```

Comment:
nit: `of the Auth0 application`

Reply:
Can you also re-order these? We prefer people use the client id + username/password or secret over the zepben token fetcher, so better to make the first thing in the list the one most relevant to users.